README.md

```diff
@@ -15,23 +16,24 @@ This library is available in **jCenter** which is the default Maven repository u
 
 ## Gradle 
 **Step 1.** Add it in your root build.gradle at the end of repositories
-```gradle
+```Gradle
 allprojects {
-		repositories {
-			...
-			maven { url 'https://jitpack.io' }
-		}
+	repositories {
+		...
+		maven { url 'https://jitpack.io' }
 	}
+}
 ```
+
 **Step 2.** Add the dependency in your apps module build.gradle
-```gradle
+```Gradle
 dependencies {
-     implementation 'com.github.manojbhadane:PaymentCardView:v1.3'
-	}
+	 implementation 'com.github.manojbhadane:PaymentCardView:v1.3'
+}
 ```
 
 ### Spread Some :heart:
-[![GitHub followers](https://img.shields.io/github/followers/manojbhadane.svg?style=social&label=Follow)](https://github.com/manojbhadane)  [![Twitter Follow](https://img.shields.io/twitter/follow/manojbhadane.svg?style=social)](https://twitter.com/Manoj_bhadane) 
+[![GitHub followers](https://img.shields.io/github/followers/manojbhadane.svg?style=social&label=Follow)](https://github.com/manojbhadane)  [![Twitter Follow](https://img.shields.io/twitter/follow/manojbhadane.svg?style=social)](https://twitter.com/manojbhadane_in) 
```

Comment:
twitter handle is incorrect on this page. Please correct it with **https://twitter.com/manojbhadane_in**